### DISCUSSION TITLE
Windows Package Manager v0.3.11102 Preview

### DISCUSSION THREAD

**Discussion by denelon** [microsoft/winget-cli#884] [Releases]
This release includes the first stable release for the import command.

The Group Policy .admx and .adml files are also included in this release.

# New Features
#220 Export/Import apps list - _Import is now a default feature_

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/winget-cli/releases/tag/v-0.3.11102-preview'>Windows Package Manager v0.3.11102 Preview</a>.</em>

**Comment by yasir-hussain2k** [comment]
AMAZING WORK !!!
